### ISSUE TITLE
Lacework Agent Helm chart contains files intended to use for development only

### ISSUE THREAD

**Issue by it-praktyk** [lacework/helm-charts#73]
The Lacework Agent Helm chart package contains

``` bash
tar -xzvf <path>/helm-charts/lacework-agent-5.5.2.tgz 
lacework-agent/Chart.yaml
lacework-agent/values.yaml
lacework-agent/values.schema.json
lacework-agent/templates/_helpers.tpl
lacework-agent/templates/access-token.yaml
lacework-agent/templates/configmap.yaml
lacework-agent/templates/daemonset.yaml
lacework-agent/templates/tests/test-agent.yaml
lacework-agent/README.md
lacework-agent/dev_install.sh
lacework-agent/dev_uninstall.sh
lacework-agent/index.yaml
lacework-agent/release_install.sh
```
All Bash files are intended for development only and shouldn't be included in the Helm package itself.

**Comment by it-praktyk** [comment]
I'm not sure if the `lacework-agent/templates/tests/` should be excluded as well.

**Comment by cirego** [comment]
The `tests` directory includes files required to validate agent installation following Helm install. They should not be excluded. Thanks!

**Comment by cirego** [comment]
Thanks again! I also merged #76 to remove the actual scripts as well.
